frontend/javascripts/viewer/model/actions/annotation_actions.ts

```diff
@@ -351,6 +354,7 @@ export const addPrecomputedMeshAction = (
   seedAdditionalCoordinates: AdditionalCoordinate[] | undefined | null,
   meshFileName: string,
   mappingName: string | null | undefined,
+  opacity: number,
```

Comment:
for the adhoc pendant, you defined a default value via `opacity: number = Constants.DEFAULT_MESH_OPACITY,`. any reason why the same isn't done here?

Reply:
So I allow undefined now and set the default when creating the action a few lines below. I hope this is fine!